tutor/src/models/tour/region.js

```diff
@@ -15,7 +14,7 @@ export default class TourRegion extends BaseModel {
   @field({ type: 'array' }) otherTours;
 
   @computed get tour_ids() {
-    return concat( [this.id], this.otherTours.peek() );
+    return this.otherTours.reverse().concat( [this.id] ).reverse();
   }
```

Comment:
Ah. yeah I think i know why.  `computeds` only update they reads observable things, and `concat` probably isn't observable.  I had that problem before when I was calling `peek` in a component's `render` and it didn't update when the values changed.  And yes it'd be nice if the tour_ids did update since if doesn't that'll probably bite us at some point.